### PR TITLE
SEARCH-459 Don't show analyzerDefinitions in ensureIndex output in cluster

### PR DESCRIPTION
**Pull request by MBkkt**
### Scope & Purpose

 Don't show analyzerDefitions in ensureIndex output in cluster

- [x] :hankey: Bugfix
- [ ] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.10: *(Please link PR)*
  - [ ] Backport for 3.9: *(Please link PR)*
  - [ ] Backport for 3.8: *(Please link PR)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

File: arangod/VocBase/Methods/Indexes.cpp [hunk 1]
```diff
@@ -497,14 +499,22 @@ Result Indexes::ensureIndex(LogicalCollection* collection, VPackSlice input,
...
-        std::string iid = tmp.slice().get(StaticStrings::IndexId).copyString();
-        VPackBuilder b;
-        b.openObject();
-        b.add(StaticStrings::IndexId,
-              VPackValue(collection->name() + TRI_INDEX_HANDLE_SEPARATOR_CHR +
-                         iid));
-        b.close();
-        output = VPackCollection::merge(tmp.slice(), b.slice(), false);
+        // we don't need analyzer revision here
+        output.openObject();
```
Comment (MBkkt) [inline arangod/VocBase/Methods/Indexes.cpp:503]:
```suggestion
        TRI_ASSERT(output.isEmpty());
        output.openObject();
```

Comment (Dronplane) [inline arangod/VocBase/Methods/Indexes.cpp:503]: Please ad an assert that output is empty (it should be, but just in case)

**Review by Dronplane — COMMENTED** [review]
LGTM in general. Please add test (js in common) that checks ensureIndex output.

File: arangod/VocBase/Methods/Indexes.cpp (hunk 1 above)
Comment (MBkkt) [reply arangod/VocBase/Methods/Indexes.cpp:504]: Done

**Review by Dronplane — APPROVED** [review]

**Pull request merged**